bin/xspec.sh

```diff
@@ -59,7 +59,7 @@ die() {
 # configured it, so there is no point to duplicate the logic here.
 # Just use it.
 
-if which saxon > /dev/null 2>&1; then
+if which saxon > /dev/null 2>&1 && saxon --help > /dev/null 2>&1 && saxon --help | grep "EXPath Packaging" > /dev/null 2>&1; then
```

Comment:
Thanks for this. I suggest to keep only the third condition to test that the saxon script exists and is the EXPath one:
```shell
if saxon --help | grep "EXPath Packaging" > /dev/null 2>&1; then
```
If the saxon script is not present the command ```saxon --help``` returns an error code which evaluates to ```false```. If a saxon script is present but it's not the EXPath packaging one, the command ```grep "EXPath Packaging"``` returns an empty result which evaluates to ```false```. Therefore I think that the other conditions ```which saxon > /dev/null 2>&1 && saxon --help > /dev/null``` are not necessary.